examples/graph3d/07_dot_cloud_colors.html

```diff
@@ -13,6 +13,9 @@
     var data = null;
     var graph = null;
 
+    function onclick(point) {
+      console.log(point);
```

Comment:
There seems to be a problem than:
![image](https://cloud.githubusercontent.com/assets/600565/21546549/be2393a4-cddf-11e6-8535-6710c95931b1.png)